packages/docs/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "@mollify/docs",
-  "version": "1.0.0",
+  "version": "0.2.1",
```

Comment:
`docs` package version number is 0.2.1, everyone else's is 0.2.11. Is there a reason for this?

Reply:
My OCD thanks you